README.md

```diff
@@ -183,6 +183,8 @@ Press `Ctrl-Shift-P` to open the command palette and type `Material Icons`.
 ## Icon sources
 
 - [Material Design Icons](https://materialdesignicons.com/)
+- [Swagger-API Github](https://github.com/swagger-api)
```

Comment:
Please remove this link to the icon, it's not necessary to be shown in the readme file.

Reply:
This has been reverted